Makefile

```diff
@@ -52,7 +52,7 @@ LDFLAGS += -extldflags "-static"
 endif
 
 .PHONY: all
-all: build test
+all: build test check
```

Comment:
check is almost as slow as test. What is your opinion about replacing test with testrace?